src/pages/BaseResultsPage.js

```diff
@@ -3,98 +3,20 @@
 import React from "react";
 import { History } from "react-router";
 import reactMixin from "react-mixin";
-import _ from "underscore";
 
 import iss from "../iss";
 import BaseCategoriesPage from "./BaseCategoriesPage";
+import ResultsListPage from "./ResultsListPage";
+import ResultsMapPage from "./ResultsMapPage";
```

Comment:
It's used in one place in Izzy, to clone an element with no significant child tree. 
A little different to cloning one of the heaviest elements as part of a page transition.